platform/pulpcore/app/viewsets/repository.py

```diff
@@ -51,6 +51,19 @@ def importers(self, request, name):
         serializer = ImporterSerializer(page, many=True, context={'request': request})
         return paginator.get_paginated_response(serializer.data)
 
+
+    def update(self, request, partial=True):
```

Comment:
I don't know how we want to document this type of thing. The ViewSet definitely will need a docstring, but it seems silly and fragile to write the same docstring twice.